packages/fx-core/src/component/error/componentError.ts

```diff
@@ -142,6 +143,18 @@ export class ExternalApiCallError extends BaseComponentInnerError {
     );
   }
 
+  static getAzureCredentialRemoteError(source: string, error?: unknown): ExternalApiCallError {
+    return new ExternalApiCallError(
+      source,
+      "GetAzureCredentialRemoteError",
+      "driver.common.FailRetrieveAzureCredentialsRemoteError",
+      HttpStatusCode.INTERNAL_SERVER_ERROR,
+      undefined,
+      ["driver.common.suggestion.retryLater"],
+      typeof error === "string" ? error : JSON.stringify(error)
```

Comment:
Based on the code, it seems `error` passed in can be in type of `string`. Why is that?